### ISSUE TITLE
[Tab Tray UI experiments] VoiceOver Announces Non-Existent Buttons and Randomly Switches Between Normal and Private Browsing

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#25833]
## Steps to reproduce
1. Open Firefox on an iPad.
1. Activate VoiceOver.
1. Navigate through the tab tray section 

#### **Actual Result:**
* The VoiceOver announces buttons that are not visible or do not exist like : selected, twelve, Private mode Large button, Synced tab large button 
* Navigation between normal and private browsing modes is random, with sudden switches between them.

#### **Expected Result:**
* VoiceOver should only announce existing buttons.
* Navigation between normal and private browsing should be consistent and predictable.

### Device & build information
* Device: iPhone 15 Pro (18.4), iPhone 14 Pro (16.2)
* Build version: v9000 (53263)

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXIOS-11860)

**Comment by data-sync-user** [comment]
➤ Laurie Marceau commented:

This ticket was merged in main and release/v138. It will normally be merged into the next weekly 137 release as well

**Comment by data-sync-user** [comment]
➤ Alina Moldovan commented:

verified as fixed using FF v9000(53622) and iPhon 16 Pro (18.3.1)

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Verified as fixed on v137.2 (53811) with iPad Mini (16.7).
